lib/libutils/isoc/bget.c

```diff
@@ -602,6 +603,48 @@ static bufsize buf_get_pos(struct bfhead *bf, bufsize align, bufsize hdr_size,
     return -1;
 }
 
+static uint64_t __maybe_unused get_free2_sum(struct bpoolset *poolset)
+{
+	struct bfhead *b = poolset->freelist.ql.flink;
+	uint64_t free2_sum = 0;
+
+	while (b != &poolset->freelist) {
+		free2_sum += b->bh.bsize * b->bh.bsize;
+		b = b->ql.flink;		  /* Link to next buffer */
+	}
+
+	return free2_sum;
+}
+
+/*
+ * update_free2_sum() - cumulative update of the free^2 sum
+ * @poolset:	The addressed poolset
+ * @rem_sz:	Size of a removed free block, 0 if unused
+ * @rem_sz2:	Size of one more removed free block, 0 if unused
+ * @add_sz:	Size of an added free block, 0 if unused
+ *
+ * As free blocks are removed, added, or merged the sum of the size ^ 2 of
+ * all free blocks needs to be updated. The most complicated case is where
+ * two free blocks are merged into one free block, both the old sizes must be
+ * supplied and the new size.
+ */
+static void update_free2_sum(struct bpoolset *poolset __maybe_unused,
+			     bufsize rem_sz __maybe_unused,
+			     bufsize rem_sz2 __maybe_unused,
+			     bufsize add_sz __maybe_unused)
+{
+#ifdef BufStats
+	uint64_t r2 = rem_sz * rem_sz + rem_sz2 * rem_sz2;
+	uint64_t a2 = add_sz * add_sz;
+	uint64_t f2s = get_free2_sum(poolset);
```

Comment:
`__maybe_unused` is required here

Reply:
Yes, I noticed that. I've force-pushed a fix.